hack/force-cert-retation.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+set -ex
+
+function usage {
+cat <<- EOF
+Usage example:
+  HANDLER_NAMESPACE=kubemacpool-system HANDLER_POD_LABEL=kubemacpool-leader=true WEBHOOK_TYPE=mutatingWebhookConfiguration WEBHOOK_CONFIG_NAME=kubemacpool-mutator WEBHOOK_SECRET_NAME=kubemacpool-service ./force-cert-rotation.sh
+
+Mandatory Arguments:
+  HANDLER_NAMESPACE     namespace of the pod operating the kube-admission's library.
+  HANDLER_POD_LABEL     specific label corresponding to the pod operating the kube-admission's library.
+  WEBHOOK_TYPE          webhook config type: [mutatingWebhookConfiguration|validationWebhookConfiguration].
+  WEBHOOK_CONFIG_NAME   webhook config name.
+  WEBHOOK_SECRET_NAME   the name of the secret created by kube-admission.
+EOF
+}
+
+function input_env_var_check {
+  # check mandatory env variables
+  if [[ -z "${HANDLER_NAMESPACE}" || -z "${HANDLER_POD_LABEL}" || -z "${WEBHOOK_TYPE}" || -z "${WEBHOOK_CONFIG_NAME}" || -z "${WEBHOOK_SECRET_NAME}" ]]; then
+    echo "Error: Not all env parameters are set."
+    usage
+    exit
+  fi
+
+  # check KUBECTL validity
+  if [[ -z "$KUBECTL" || -z "$($KUBECTL version)" ]]; then
+    echo "Error: KUBECTL is not defined"
+    exit
+  fi
+}
+
+# input parameter parsing
+function input_param_parse {
+  while [[ ! -z "$1" ]]; do
+    case "$1" in
+      --help)
+          usage
+          exit
+          ;;
+      *)
+          break
+          ;;
+      esac
+      shift
+  done
+}
+
+# retrive tls.crt from secret data
+function get_secret_data {
+  echo "$(${KUBECTL} get secret -n ${HANDLER_NAMESPACE} ${WEBHOOK_SECRET_NAME} -oyaml | shyaml get-value data)"
+}
+
+function eventually {
+    timeout=100
+    interval=10
+    cmd=$@
+    echo "Checking eventually $cmd"
+    while ! $cmd; do
+        sleep $interval
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 1
+        fi
+    done
+}
+
+function isSecretRotated {
```

Comment:
is_secret_rotated, let's be consistent, also pass OLD_SECRET_DATA as an argument, so it's clear were stuff comes frome.